app/components/post_header/post_pre_header.js

```diff
@@ -64,19 +63,19 @@ export default class PostPreHeader extends PureComponent {
             <View style={[style.container, (isConsecutive && style.consecutive)]}>
                 <View style={style.iconsContainer}>
                     {isPinned && !skipPinnedHeader &&
-                    <Image
-                        id='pinIcon'
-                        source={pinIcon}
+                    <CompassIcon
+                        name='pin-outline'
+                        size={14}
                         style={style.icon}
                     />
                     }
                     {isPinnedAndFlagged &&
                     <View style={style.iconsSeparator}/>
                     }
                     {isFlagged && !skipFlaggedHeader &&
-                    <Image
-                        id='flagIcon'
-                        source={flagIcon}
+                    <CompassIcon
+                        name='flag-outline'
```

Comment:
are we changing the flag for **save**  ?

Reply:
Save / Unsave is correct